src/dotnet/commands/dotnet-run/LocalizableStrings.cs

```diff
@@ -16,26 +16,14 @@ internal class LocalizableStrings
 
         public const string RunCommandException = "The build failed. Please fix the build errors and run again.";
 
-        public const string RunCommandMSBuildExtensionsPath = "MSBuildExtensionsPath";
-
-        public const string RunCommandConfiguration = "Configuration";
-
-        public const string RunCommandTargetFramework = "TargetFramework";
-
-        public const string RunCommandProjectInstance = "RunCommand";
-
-        public const string RunCommandOutputType = "OutputType";
+        public const string RunCommandExceptionUnableToRun = "Unable to run your project\nPlease ensure you have a runnable project type and ensure '{0}' supports this project.\nThe current {1} is '{2}'";
```

Comment:
@akoeplinger Concatenation of multiple strings is something to be avoided in localization. I'm unaware of `\n` causing issues in this kind of usage.